setup.cfg

```diff
@@ -14,9 +14,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-[egg_info]
-tag_build = dev
-
```

Comment:
@IlyaFaer what is this for and is it ok that I removed it? I don't want this to be marked as a dev build (or for the versions to have any suffix).

Reply:
Yeah, it can be deleted